packages/templates/clients/websocket/python/README.md

```diff
@@ -3,18 +3,47 @@
 
 You can test this template:
 1. Clone the project and run `npm install`
-2. Navigate to `packages/templates/clients/websocket/python`
-3. Install with `npm install` and run test with `npm run test`
-4. Go to test folder with `cd test`
-5. Install dependencies of the generated client: `pip install -r temp/snapshotTestResult/custom_client_hoppscotch/requirements.txt`
-6. Start example script that uses a client library generated by the test: `python example.py`
+1. Navigate to `packages/templates/clients/websocket/python`
+1. Install with `npm install` and run test with `npm run test`
+1. Install dependencies of the generated client: `pip install -r test/temp/snapshotTestResult/custom_client_hoppscotch/requirements.txt`
+1. Start example script that uses a client library generated by the test: `python example.py`
 
 > By default this is testing against Hoppscotch echo service. You can modify `packages/templates/clients/websocket/python/example.py` and change line 7 to `from temp.snapshotTestResult.client_postman.client import PostmanEchoWebSocketClientClient` and line 21 `client = PostmanEchoWebSocketClientClient()` and run `python example.py` again. You will see example still works but agains different API. This is possible as both AsyncAPI documents have the same name of operation for sending messages: `sendEchoMessage` so each client generated has the same API.
 
 ## Client for Slack
 
 To run the Slack Client example, follow the steps above but with 2 exceptions:
 - Use `example-slack.py` instead of `example.py`.
-- You need to pass environment variables with proper authorization details. Your command must look like this: `TICKET={provide secret info} APP_ID={provide id of the slack app} python example-slack.py`
+- You need to pass environment variables with proper authorization details. Your command must look like this: `TICKET=1d967f38-ccff-44f6-adec-9616eec9c4b7 APP_ID=00dfdcccb53a2645dd3f1773fcb10fa7b0a598cf333a990a9db12375ef1865dd python example-slack.py`
```

Comment:
If we have token in pinned item in `#generator` channel in Slack, I suggest to revert changes to old one:

```
TICKET={provide secret info} APP_ID={provide id of the slack app} python example-slack.py
```

What do you think?